migration/sql-files/002-oauth-states.sql

```diff
@@ -3,6 +3,6 @@ CREATE TABLE oauth_state_references (
     created_at timestamp with time zone,
     updated_at timestamp with time zone,
     deleted_at timestamp with time zone,
-    id uuid primary key DEFAULT uuid_generate_v4() NOT NULL,
+    id varchar primary key DEFAULT CAST(uuid_generate_v4() AS varchar(50)) NOT NULL,
```

Comment:
Don't change existing .sql migration files. Each .sql file represents particular DB version. Add a new .sql to migrate DB to a new version.